ios/FluentUI/Vnext/Popup Menu/PopupMenuItemCell.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+// MARK: - State
+
+public class PopupMenuItemCellState: MSFListCellState {
+
+    // Set/Unset selection for item cell
+    @objc @Published public var isSelected: Bool = false {
+        didSet {
+            if isSelected {
```

Comment:
What are the results of the experimentation we discussed offline about doing this in the SwiftUI layer?
Are we going to be able to define this logic over there?

Reply:
yes, I will put another PR today it today.